modules/magnolia/src/test/scala/pureconfig/module/magnolia/CoproductHintSuite.scala

```diff
@@ -76,7 +76,9 @@ class CoproductHintSuite extends BaseSuite {
       val cc = implicitly[ConfigConvert[Conf]]
 
       val conf = ConfigFactory.parseString("{ typ = ambiguous-conf }")
-      cc.from(conf.root()) should failWithType[KeyNotFound] // "typ" should not be passed to the coproduct option
+      cc.from(conf.root()) should failWithReason[
+        KeyNotFound
+      ] // "typ" should not be passed to the coproduct option
```

Comment:
nit: let's move the comment to a separate line to avoid scalafmt formatting it like this?

Reply:
I think this was due to a previous rename of that method. The original line should fit without line breaks.